assets/js/components/App.tsx

```diff
@@ -28,6 +29,7 @@ class AppRoutes extends React.Component {
           <Route path="*" element={<Dashboard />}>
             <Route path="dashboard" element={<PlacesPage />}></Route>
             <Route path="alerts" element={<AlertsPage />}></Route>
+            <Route path="alerts/:id" element={<AlertDetails />}></Route>
```

Comment:
Just trying to understand React Router and outlet a bit. Is this bit of code here how the child components know where to go to fetch state when we call `useDashboardContext`?

Reply:
The state retrieved from `useDashboardContext` is defined in `Dashboard`. It is passed into the `<Outlet />` component. `Outlet` is basically a short way of rendering a child route's component on the parent route. So `Dashboard` is the parent route and when navigating to `/alerts`, it will replace `<Outlet />` with `<AlertsPage />`.